style/layer/poi.js

```diff
@@ -0,0 +1,82 @@
+import * as label from "../constants/label.js";
+
+const poiIcon = [
+  "match",
+  ["get", "class"],
+  "bar",
+  "martini-glass",
+  "beer",
+  "martini-glass",
+  "hospital",
+  "hospital-cross",
+  "dot_city",
+];
+
+const poiFilter = [
+  "any",
+  ["==", ["get", "class"], "bar"],
+  ["==", ["get", "class"], "beer"],
+  ["==", ["get", "class"], "hospital"],
+];
+
+const iconColor = [
+  "match",
+  ["get", "class"],
+  "bar",
+  "#0000dd",
+  "beer",
+  "#0000dd",
+  "hospital",
+  "#ff0000",
+  "#000000",
+];
+
+const labelPaint = {
+  "text-halo-color": "#ffffff",
+  "icon-halo-color": "#ffffff",
+  "text-halo-width": 1.5,
+  "icon-halo-width": 1.5,
+  "text-halo-blur": 1,
+  "icon-halo-blur": 1,
+  "text-color": iconColor,
+  "icon-color": iconColor,
+};
+
+export const poi = {
+  id: "poi",
+  type: "symbol",
+  paint: labelPaint,
+  filter: poiFilter,
+  layout: {
+    "text-font": ["Metropolis Bold"],
```

Comment:
Can we get away with using a regular typeface for these labels, or would they get lost in a crowd of road labels? It would be nice to reserve boldface labels for things we want to highlight, like places and landmarks that are especially important irrespective of their `class`.

Reply:
Looks like we can use "Metropolis Light" though ultimately I'd like to update the fonts used in this style more broadly to not be handcuffed to the few options we have on the openmaptiles fonts server.

Reply:
#393 will probably force us to migrate to another font.